packages/lesswrong/components/posts/PostsItemNewDialogueResponses.tsx

```diff
@@ -0,0 +1,26 @@
+import { gql, useQuery } from "@apollo/client";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import React from "react";
+
+const PostsItemNewDialogueResponses = ({postId, unreadCount} : {postId: string, unreadCount: number}) => {
+  
+  const { ContentItemBody, Loading, NoContent }= Components
+
+  const { data, loading } = useQuery(gql`
+    query LatestDialogueMessages($dialogueId: String!, $unreadCount: Int!) {
+      latestDialogueMessages(dialogueId: $dialogueId, numMessages: $unreadCount)
+    }
+  `, {variables: {dialogueId: postId, unreadCount  }});
+
+  return loading ? <Loading /> : data ? data.length ? <ContentItemBody
+    dangerouslySetInnerHTML={{__html: data.latestDialogueMessages.join('')}} />
```

Comment:
Is this actually safe?  Maybe the html contents of a dialogue are already necessarily sanitized in such a way that it's safe to just stick them directly into `dangerouslySetInnerHTML` like this (even when concatenating them), but not 100% sure.  @jimrandomh do you know?

Reply:
I believe this is safe and Habryka thought it should be too; going to count this resolved